frc.bat

```diff
@@ -1,28 +1,25 @@
 @echo off
 
-:frc_help
-echo Usage: frc.cmd [verify ^| deploy ^| console ^| shell ^| rlds]
-goto :eof
-
 if "%1" == "" (
-    call :frc_help
+    echo Usage: frc.bat [verify | deploy | console]
 ) else if "%1" == "verify" (
```

Comment:
This whole thing can just be replaced with `if "%1" == "verify" (`. If no argument is supplied the code execution will fall through to the help branch anyway.